src/main/java/ro/societateahermes/backendservice/controller/controllerImplementation/FormController.java

```diff
@@ -0,0 +1,53 @@
+package ro.societateahermes.backendservice.controller.controllerImplementation;
+
+import org.springframework.web.bind.annotation.*;
+import ro.societateahermes.backendservice.entities.DTO.FormDto;
+import ro.societateahermes.backendservice.entities.form.Form;
+import ro.societateahermes.backendservice.service.serviceImplementation.FormService;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/form")
+public class FormController {
+    private final FormService formService;
+
+    public FormController(FormService formService) {
+        this.formService = formService;
+    }
+
+    @GetMapping
+    public List<FormDto> getAll() {
+        List<Form> forms = formService.getAll();
+        return forms.stream()
+                .map(this::convertToDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping
+    public void save(@RequestBody FormDto formDto) {
+        formService.save(this.convertToEntity(formDto));
+    }
+
+    @DeleteMapping("/{formId}")
+    public void delete(@PathVariable("formId") Long formId) {
+        formService.delete(formId);
+    }
+
+    private FormDto convertToDto(Form form) {
```

Comment:
Metodele astea nu ar trebui sa apara in controller. Daca le pui aici, atunci controllerul va capata doua responsabiitati, aceea de aprocesa requesturi din frontend si acela de a converti din dto in entitate si invers, ceea ce nu-i bine. Eu daca voi vrea sa schimb vreodata controllerul va trebui sa rescriu si metodele astea doua, chiar daca nu am nevoie. Pune-le intr-un obiect separat. Poate sa-l cheme formMapper sau ceva de genul. Sa pui metode de genul intr-o clasa care nu are nicio treaba cu ele e bad design